packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -43,29 +50,43 @@ class ChromeDriverInstaller {
         await runDriver();
       }
     } finally {
-      driverDownload?.deleteSync();
+      // Only delete if the user is planning to override the installs.
+      // Keeping the existing version might make local development easier.
+      if (!alwaysInstall) {
+        driverDownload?.deleteSync();
+      }
     }
   }
 
-  Future<bool> install() async {
-    if (!isInstalled) {
+  Future<bool> install({bool alwaysInstall = false}) async {
```

Comment:
This method always returns `true` (because `_installDriver` always returns `true`). Maybe make them all `void`?

Reply:
I think it is a very good point. I did what we did in the previous PR! I changed everything to crash (throw exception) or void.  It cleaned up a lot of code, thanks.